### DISCUSSION TITLE
Heal / revive commands don't properly heal terminal husk infection

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13771] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you have <75% husk infection  and try to heal with commands the infection will be gone but you will still have tentacles in your mouth, pressure immunity and will turn into husk if you die, ~and there's no cure to this xd~ , actually no, save-loading might help

Not really an issue (unless you are a filthy cheater), but it might interfere with testing at some point

### Reproduction steps

giveaffliction "Husk infection" 80
revive
kill
oops, you are a husk

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I wouldn't necessarily consider this a bug: there's no way to heal the affliction at the late stages and that seems to extend to console commands. I feel it'd be just add unnecessary complexity to add support for removing the limbs and "reversing" the transformation for the purpose of a console command you can't use during normal gameplay.
